src/resources/assets/scss/customs/_list.scss

```diff
@@ -0,0 +1,335 @@
+// DataTables

```

Comment:
I'm still getting this empty column on Articles CRUD on iPhoneX:

![Screenshot 2021-12-27 at 09 00 06](https://user-images.githubusercontent.com/1032474/147444102-041f36ae-3d0f-42f6-8eef-2c1e54e7bda0.png)

Which I think is a big usability problem. While we're at it, we should fix this too. If we can't fix it with CSS... I've discovered setting the `CRUD::column('title')->limit(30)` fixes it. Perhaps a reasonable solution would be to change the default `limit` for columns from 40 to 30? It's not ideal... but I think this bug is important enough for us to do that, if needed.

So if you can't find a CSS solution for this @promatik ... let's change the limit so we make sure all columns are visible on most mobiles.

Reply:
@tabacitu this is an hard one 😕
I tried a bunch of css solutions, and I couldn't find a way to resize the columns to match the criteria of datatables to allow the columns to be resized.
I'm out of ideas, and I don't like updating the limit from 40 to 30 😕
Do you think we should go that way?

Reply:
I've created a separate issue for this https://github.com/Laravel-Backpack/CRUD/issues/4049 - it's not exactly a new issue, it's not introduced by this PR, so let's talk about it separately. This will allow me to merge this PR now.